Use the northendlab theme

koheiw opened the pull request.

kbenoit commented:
(Based on https://deploy-preview-60--hardcore-cray-f31080.netlify.com/)

I like the clean look and the layout and fonts. If the code is clean and modern and lets us add the R-bloggers blogroll etc, great. Some comments for improvement or revision follow. I know this is more critique than suggesting constructive alternatives, so if I can do more to help actively, let me know.

## Overall look

The cleanliness is great. But the overall theme is devoid of contrast, with just white, black/gray, and shades of blue. The text fonts are also a lighter gray on white, and this makes it hard to read. 

Perhaps zones of colour, as we had before? Just one or two?

I think we need either larger and more visible menu fonts, or to stop the bleed of the wordcloud image into the menu. 

I don't really like the text boxes, which seem narrow and make me think I accidentally clicked on Tablet mode in my Chrome/Developer tools. Not sure we need the shadow look either. What alternatives can we have in this theme? 

## Content

Front page: I think this should describe what QI does and have some links, not start with the news.  We could integrate a preview of the news into the front page, but it should not be the main element, which should describe the QI and what it's for.

The main text headline should say: "Promoting open-source solutions to quantitative text analysis" not focusing on the legal structure of the organization. (We can describe that in the text I am suggesting we add above. Technically, it's a Community Interest Company, not a non-profit, but there are limits on what can be done with any profits. This is the same as RStudio now!)

We could reorganize About into three sub-menus: Purpose, People (move it to there), Company. This is similar to what we have in the older site.

I like having photos (again) in the People page but I'm not stuck on that.

For Projects: what if make this into sub-pages, such as:
- packages, listing each package with some image or logo for each? We can think about redesigning the quanteda logo and adding logos for the other packages consistent with the new quanteda one. I can get a designer to help if we want.
- documentation - here we list the tutorial site, future learnr sites, videos

or maybe we need a separate menu for Learning and put those below it

## Minor

- add YouTube social media button
- need to match/integrate blog.quanteda.org once we decide on the main site

koheiw commented:
Thanks for the comments. I made following changes:

- Text colors: it is darker per your request
- Page width: made wider in non-news pages.
- Top page banner: adjusted padding and changed the text
- Footer: added YouTube link

However, I kept the template's look largely intact, because the extreme simplicity is the feature of this graphic design and only edgy websites look good in one or two years in the future (let's bet on this). If we don't like the extreme simplicity, we should find a different template.

We can describe what QI is on the top page in a box like in the demo's ["Get the best hugo templates"](https://themes.gohugo.io/theme/northendlab-hugo/), but I don't think we need it, because people can click "About" in the menu straightaway.

We should update the content of the pages, but I don't think we need sub-pages because pages are still short (this template does not have sub-menus either). I also think it is better to exclude promotional content, which should be in separate websites with proper landing page structure, from this website.

I did not insert images into pages to make this website purely *text-based*.

kbenoit commented:
OK, looking better. I can live with the look, although it might not have been my first choice. I like http://demo.themefisher.com/gosaas-hugo/ better (with less bright colours, if you prefer). Is it very complicated to preview what you've done but using this as a theme?

The news page is very minimal and does not have tags, searching, summaries, etc. (as for instance https://blog.quanteda.org/ has). Is this because these are turned off, or because the theme cannot do them? If the latter, and we want to use this same look for the blog, then I think it's too minimal. Also why not match the News width to that of the other pages too? It's a responsive theme so will adjust if the window is smaller, so I see no reason not to match their sizes.

I still don't like the landing page to be News, since that's never been the focus of the QI website anyway. But it would make a nice thing to include a summary of, like many web pages can, on the front page but being more fully listed on the News page. The main purpose of the QI page is to:
    - promote its purpose and activities, for attracting talent, funding, and attention. This is why explaining clearly what its purpose is and linking to its other aspects is the most important function of the page.
    - explain the ecosystem of supported activities, including an overview of the packages, the learning materials, and the services/training activities. This will be really important when we start offering paid training for instance.
    - provide news on events (our News section)
    - link to the blog which has more detailed, technical analyses

Why would we want a text-only page? Some of the nicest parts of the existing page are the images. For a Hugo with images of the appropriate sizes, this will not affect performance. Is this some new aesthetic? There are ways to be minimal but still allow judicious use of images. I think the News/Events would be nice to have images accompany them. Also we need images for things like the acknowledgements, or screenshots, etc. People images make sense too!

stefan-mueller commented:
Thanks a lot for your work on the website, @koheiw. 

I agree with @kbenoit's comments. The page should serve the purpose of giving users an overview of what we are offering (package development, training events, other activities) and promote our "product". This is especially important in order to attract potential training clients. Thus, we will need a front page that summarises these goals clearly. I also think that the website should not be text-only. The university logos and acknowledgments, for instance, were really important on the old landing page, and we should keep both. I also like http://demo.themefisher.com/gosaas-hugo/ as a template.

Once we have agreed on the layout and structure of the new website, I could write a draft for the new landing page, explain the ecosystem (overview of the packages, the learning materials, and the services/training activities), continue updating the news section, and administer the blog.

koheiw commented:
@stefan-mueller If you want to add information about workshop you should make pages in current website, becasue we (I or other designers) decided the structure of new website based on what we see. I proposed the website based on my understanding of how the website has been used for two years (almost never updated).

I also think it was a mistake to mix promotion materials and corporate information in one website. It is better to make a separate website for training courses.

I like http://demo.themefisher.com/gosaas-hugo/ too but we should always think what graphics (illustrations and pictures) we can use for our website in terms of licenses and themes. It is really difficult for us to find nice graphics actually.
